platform/util/platform-util-unavailableurls/src/main/java/org/codice/ddf/platform/util/http/UnavailableUrls.java

```diff
@@ -24,15 +24,25 @@
 import org.slf4j.LoggerFactory;
 
 /**
+ * /**
  * A class to contain a set of unavailable URL's. They will be periodically checked
  * to determine if the URL has become available. If the URL is reachable, this implies
  * the web service is available, and the URL will be removed from the set.
  * <p>
- * NOTE: to avoid excessive network traffic, the period to wait before checking for a
+ * NOTE:
+ * to avoid excessive network traffic, the period to wait before checking for a
  * URL increases exponentially.
  */
 public class UnavailableUrls {
 
+    // Name of system property
+    static final String MAX_TIMEOUT_SECONDS_PROPERTY =
+            "org.codice.ddf.platform.util.http.maxRetryInterval";
+
+    // Name of system property
+    static final String INITIAL_TIMEOUT_SECONDS_PROPERTY =
```

Comment:
I'm not sure `min` is any more clear than `initial`, but I feel like symmetric wording helps make the relationship more clear to users.